Purchases/SwiftSources/LocalReceiptParsing/LocalReceiptParser.swift

```diff
@@ -12,12 +12,17 @@ import Foundation
          UnknownError
 }
 
+private enum IntroEligibilityStatus: Int {
+    case unknown,
+         eligible,
+         ineligible
+}
+
 @objc public class LocalReceiptParser: NSObject {
+    
     @objc public func checkTrialOrIntroductoryPriceEligibility(withData data: Data,
                                                                productIdentifiers: [String],
-                                                               completion: ([String : RCIntroEligibility], Error?) -> Void) {
-        completion([:], NSError(domain: "This method hasn't been implemented yet",
-                                code: LocalReceiptParserErrorCode.UnknownError.rawValue,
-                                userInfo: nil))
+                                                               completion: ([String : NSNumber], Error?) -> Void) {
+        completion(["test": NSNumber(value: IntroEligibilityStatus.unknown.rawValue)], nil)
```

Comment:
I don't know swift but do you need the `NSNumber` here?

Reply:
no... not for swift, anyway. But once this gets converted to Obj-c, NSDictionaries can't hold raw types 🙄, so I was wrapping it in an NSNumber. 
come to think about it, maybe this isn't needed if the conversion from swift to obj-c does this automatically, I haven't tested.

Reply:
just tried it out and it loks like it does automatically get translated to NSNumber when in objective-c land.